cmd/poktrolld/cmd/config.go

```diff
@@ -35,17 +35,26 @@ func initSDKConfig() {
 	config.Seal()
 }
 
-// initCometBFTConfig helps to override default CometBFT Config values.
-// return cmtcfg.DefaultConfig if no custom configuration is required for the application.
+// The code below changes how the default configuration files are rendered on `poktrolld init` command. This command
+// is often used by the validator and full node runners to provision the configuration prior to starting the node.
+// These are the values **WE WANT** participants to have. This doesn't guarantee the node runners won't adjust the values
+// but it helps making sure most of them are using the following configuration.
+// Worth noting that changing these values in the future won't magically update them in the existing configuration,
```

Comment:
I'm not completely sure what you mean by this sentence. Please update or delete.

Reply:
Pushed the new version that I think better explains what's going on.

I'll merge it, but if you don't like it for some reason, please let me know and I'll address.

Thank you for your review!